Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1116,6 +1116,12 @@ namespace Babylon
 
     void NativeEngine::CopyTexture(const Napi::CallbackInfo& info)
     {
+        if (!info[0].IsExternal() ||
+            !info[1].IsExternal())
+        {
+            return;
+        }
+
```

Comment:
This is a bit weird. We just silently do nothing when the types are wrong? Is that the intended behavior?

Reply:
more was a check because the external wasn't unwrapping correctly for jsi when it was a unique struct definition. I'll undo this check.

Reply:
i'm going to keep this check but will switch to throwing instead of returning

Reply:
this should now be fixed.